CODEOWNERS

```diff
@@ -3,31 +3,31 @@
 # Do not use wildcard on all source yet
 # *                                        @galak @nashif
 
-.known-issues/*                          @inakypg @nashif
+.known-issues/                           @inakypg @nashif
```

Comment:
Not sure if this changes was intended to be part of this PR about documentation.